bids/grabbids/__init__.py

```diff
@@ -1,3 +1,7 @@
-from .bids_layout import BIDSLayout
-from .bids_validator import BIDSValidator
+import warnings
+from ..layout.bids_layout import BIDSLayout
+from ..layout.bids_validator import BIDSValidator
 __all__ = ["BIDSLayout", "BIDSValidator"]
+
+warnings.simplefilter('always', DeprecationWarning)
+warnings.warn("grabbids has been renamed to layout", DeprecationWarning)
```

Comment:
Could we note in the warning text the version we're deprecating at (0.6.5 or 0.7), and when to expect its removal? This both helps downstream projects plan a transition path and lets pybids developers know when they can fully remove it.

Reply:
How's this for a proposed change to those lines:
``` diff
-warnings.simplefilter('always', DeprecationWarning)
-warnings.warn("grabbids has been renamed to layout", DeprecationWarning)
+warnings.warn("grabbids has been renamed to layout in version 0.6.5, and will be removed in version 0.7", FutureWarning)
```
I could change the versions to whatever you think is best. `FutureWarning` shows up by default without changing the warning filter. I could also do that in another way

Reply:
@tyarkoni Will the next version be 0.6.5 or 0.7?

And I would say we should remove in 1.0, unless that's expected within the next month. (I realize pybids is alpha, but the grabbids API has been around for a long time.)